FlaskApp/web_app/templates/index.html

```diff
@@ -28,6 +28,11 @@
         <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
     <![endif]-->
     <script src="https://code.jquery.com/jquery-2.1.0.min.js" integrity="sha256-8oQ1OnzE2X9v4gpRVRMb1DWHoPHJilbur1LP9ykQ9H0=" crossorigin="anonymous"></script>
+    <script type="text/javascript" src="../static/js/bootstrap.js"> </script>
+    <script src="http://code.jquery.com/jquery-1.9.1.js"></script>
+    <script   src="https://code.jquery.com/jquery-1.9.1.min.js"   integrity="sha256-wS9gmOZBqsqWxgIVgA8Y9WcQOa7PgSIX+rPA0VL2rbQ="   crossorigin="anonymous"></script>
+      <script   src="https://code.jquery.com/ui/1.10.0/jquery-ui.min.js"   integrity="sha256-hTpbeVXhgCmfO7nGcWp9d1kImKbzY6gN0Vo5u5wLuss="   crossorigin="anonymous"></script>  
+
     
     
        <script type="text/javascript">
```

Comment:
generally. what we would do is to put all JS in a JS file and include in html header. just like above.